app/views/Main.js

```diff
@@ -68,11 +57,16 @@ export const Main = () => {
     };
   }, [navigation, updateStateInfo]);
 
+  const handleBackPress = () => {
+    BackHandler.exitApp(); // works best when the goBack is async
+    return true;
+  };
+
   if (!trackingInfo.canTrack) {
-    return <TracingOffScreen />;
+    return <TracingOffScreen onPress={openSettings} />;
```

Comment:
If this is only being used for GPS, we no longer need to pass an onPress to the TracingOffScreen. We can import this directly in that screen.